Common/src/CConfig.cpp

```diff
@@ -6824,7 +6855,27 @@ void CConfig::SetOutput(SU2_COMPONENT val_software, unsigned short val_izone) {
 
   if (val_software == SU2_COMPONENT::SU2_CFD) {
 
-    cout << "Writing solution files every " << VolumeWrtFreq <<" iterations."<< endl;
+    if (nVolumeOutputFrequencies != 0) {
+      cout << "File writing frequency: " << endl;
```

Comment:
It seems that OUTPUT_FILES= NONE is a valid option, although it does not exist in the enum class. This causes an issue for an FSI testcase, where this option is used in the config file. I will add NONE as an option to explicitly allow the user to say that you do not want output files.

Reply:
NONE is an option for all lists, almost every COption allows NONE and returns early.
What is the issue? Please don't add code to handle none explicitly if the number of files is zero why would there be a problem?...

Reply:
We have a number of makepairs with NONE explicitly defined, so is there a way to do something like 
`if (kindturbmodel != TURB_MODEL::NONE)`
without defining a makepair for it? Then we can get rid of a bunch of makepairs.

Reply:
no, the purpose of having enum class, is precisely to avoid that.